.gitlab.mk

```diff
@@ -134,6 +134,10 @@ deploy: export LD_LIBRARY_PATH=/usr/local/lib
 deploy:
 	echo ${GPG_SECRET_KEY} | base64 -d | gpg --batch --import || true
 	case "${GITHUB_REF}" in                                       \
+	refs/tags/*-alpha*|refs/tags/*-beta*|refs/tags/*-rc*) \
```

Comment:
The commit message contains a mention of the `*-dev*` tag. It looks as a mistake.

Reply:
Fixed